Lookuptable/testPages/data/export_data_page.py

```diff
@@ -0,0 +1,489 @@
+import os
+import time
+import pandas as pd
+from datetime import datetime, timedelta
+
+from common_utilities.selenium.base_page import BasePage
+from common_utilities.path_settings import PathSettings
+from HQSmokeTests.userInputs.user_inputs import UserData
+from selenium.common.exceptions import StaleElementReferenceException, TimeoutException, \
+    ElementClickInterceptedException, NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+""""Contains test page elements and functions related to the exports module"""
+
+
+def latest_download_file():
+    os.chdir(PathSettings.DOWNLOAD_PATH)
+    files = sorted(os.listdir(os.getcwd()), key=os.path.getmtime)
+    newest = max(files, key=os.path.getctime)
+    print("File downloaded: " + newest)
+    return newest
+
+
+class ExportDataPage(BasePage):
```

Comment:
Same! Redundant code. We can reference https://github.com/dimagi/dimagi-qa/blob/master/HQSmokeTests/testPages/data/export_data_page.py

Reply:
Changes made. Commit code for reference : fccca641344bce18df6c10d99660adae9e687091